docs/reference/query-languages/esql/esql-multi-index.md

```diff
@@ -135,6 +135,60 @@ FROM events_*
 | 2023-10-23T12:27:28.948Z | 172.21.2.113 | 2764889 | Connected to 10.1.0.2 |
 | 2023-10-23T12:15:03.360Z | 172.21.2.162 | 3450233 | Connected to 10.1.0.3 |
 
+### Date and date_nanos union type [esql-multi-index-date-date-nanos-union]
+
```

Comment:
``````suggestion
```{applies_to}
stack: ga 9.2.0
```
``````

Reply:
@fang-xing-esql forgot that we need to version tag new features :)

[applies_to](https://elastic.github.io/docs-builder/syntax/applies/) helps us do that